contracts/HtsSystemContract.sol

```diff
@@ -48,6 +49,265 @@ contract HtsSystemContract is IHederaTokenService, IERC20Events, IERC721Events {
         (responseCode, tokenInfo) = IHederaTokenService(token).getTokenInfo(token);
     }
 
+    function getNonFungibleTokenInfo(address token, int64 serialNumber)
```

Comment:
nit: let's sort methods as they are declared in `IHederaTokenService`. This will improve readability, as the same time it will group related methods together, _e.g._, `associateTokens` and `associateToken`.